### DISCUSSION TITLE
What can be done to make it clear that HyperFormula has no GUI?

### DISCUSSION THREAD

**Discussion by sequba** [handsontable/hyperformula#971] [General]
Most of the examples in HyperFormula docs presents the date in a table. Some of the users are confused by that. They think that this table is a part of HF.

What can be done to make it clear for the readers of our documentation that HyperFormula has no GUI?

**Comment by sequba** [comment]
This might help: #972

**Comment by warpech** [comment]
I think that all the examples, should emphasize that HyperFormula is a library by either using:

1. interactive code runner where there is a "code box" and a "console box", for example using https://runkit.com/docs/embed
2. or, simple GUI with just the code snippet, input field and an output div

We would make only one exception on the page called Demo that we have now at https://handsontable.github.io/hyperformula/guide/demo.html

Need to be discussed with @krzysztofspilka
